voice/bxml/verbs/startRecording.md

```diff
@@ -2,31 +2,33 @@
 | password                     | (optional) The password to send in the HTTP request to `recordingAvailableUrl` or `transcriptionAvailableUrl`. If specified, the URLs must be TLS-encrypted (i.e., `https`).                                           |
-| tag                          | (optional) A custom string that will be sent with this and all future callbacks unless overwritten by a future `tag` attribute or cleared.<br><br>May be cleared by setting `tag=""`<br><br>Max length 256 characters. |
+| tag                          | (optional) A custom string that will be sent with this and all future callbacks unless overwritten by a future `tag` attribute or cleared.<br><br>May be cleared by setting `tag=""`<br><br>Max length 256 characters.<br><br>Note: this option will be silently ignored if set on a conference recording, and the tag associated with the conference (if any) will be sent. |
 | fileFormat                   | (optional) The audio format that the recording will be saved as: `mp3` or `wav`.  Default value is `wav`.                                                                                                              |
-| multiChannel                 | (optional) A boolean value indicating whether or not the recording file should separate each side of the call into its own audio channel. Default value is `false`.                                                    |
+| multiChannel                 | (optional) A boolean value indicating whether or not the recording file should separate each side of the call into its own audio channel. Default value is `false`.<br><br>Note: this option will be silently ignored if set on a conference recording; only single-channel recordings are allowed on conferences |
 
 If the `recordingAvailableUrl` attribute is specified, then the [Recording Available](../callbacks/recordingAvailable.md)
-event is sent to the URL once the recording is available for download, indicating the `mediaUrl` and if there was any issue processing the recording.
+or [Conference Recording Available](../callbacks/conferenceRecordingAvailable.md) event is sent to the URL once the
+recording is available for download, indicating the `mediaUrl` and if there was any issue processing the recording.
 BXML returned in response to this callback will be ignored.
 
-If the `transcriptionAvailableUrl` attribute is specified, then the [Transcription Available](../callbacks/transcriptionAvailable.md)
+If the `transcriptionAvailableUrl` attribute is specified for a call recording, then the [Transcription Available](../callbacks/transcriptionAvailable.md)
 event is sent to the URL once the transcription is available for download.
-BXML returned in response to this callback will be ignored.
+BXML returned in response to this callback will be ignored. Note that this attribute will be silently ignored if recording a conference.
```

Comment:
Worth sticking to the same `<br>br>Note:` style you used in the other attributes?

Reply:
Oh this is not in the table, nvm.